index.js

```diff
@@ -12,3 +12,9 @@ I need this code, just don't know where, perhaps should make some middleware, do
 
 Go code!
 */
+
+const server = require('./server');
+
+server.listen( 5000, () => {
```

Comment:
start getting in the habit of defining a port and passing that to the listen method so you dont need to refactor when you want to deploy... const port = process.env.PORT || 5000  server.listen(port, () => console.log........